middleware.js

```diff
@@ -108,6 +110,12 @@ module.exports = function(options) {
         sassDir = dirname(sassPath);
       }
 
+      if (sourceMap === true) {
+        var cssMapPath = cssPath + '.map';
+      } else if (typeof sourceMap === 'string') {
+        var cssMapPath = join(dirname(cssPath), sourceMap);
+      }
+
```

Comment:
Ops, I thought you are setting this for upstream option.
Disregard my previous comment.